lib/babel-build.js

```diff
@@ -18,6 +18,7 @@ function babelOptions(libraryName, _options) {
       'es6.constants',
       'es6.modules'
     ],
+    externalHelpers: true,
```

Comment:
this is from implementing a plugin to strip classCallCheck while investigating alternative adapters, it should be removed.